newIDE/app/src/AssetStore/BehaviorStore/BehaviorStoreContext.js

```diff
@@ -216,16 +199,46 @@ export const BehaviorStoreStateProvider = ({
   const allTranslatedBehaviors = React.useMemo(
     () => {
       const allTranslatedBehaviors: {
-        [name: string]: BehaviorShortHeader | SearchableBehaviorMetadata,
+        [name: string]: BehaviorShortHeader,
       } = {};
       for (const type in translatedBehaviorShortHeadersByType) {
         allTranslatedBehaviors[type] =
           translatedBehaviorShortHeadersByType[type];
       }
       for (const installedBehaviorMetadata of installedBehaviorMetadataList) {
+        const repositoryBehaviorMetadata =
+          translatedBehaviorShortHeadersByType[installedBehaviorMetadata.type];
+        const behaviorMetadata = repositoryBehaviorMetadata
+          ? {
+              isInstalled: true,
+              tier: repositoryBehaviorMetadata.tier,
+              version: repositoryBehaviorMetadata.version,
+              url: repositoryBehaviorMetadata.url,
+              headerUrl: repositoryBehaviorMetadata.headerUrl,
+              extensionNamespace: repositoryBehaviorMetadata.extensionNamespace,
+              authorIds: repositoryBehaviorMetadata.authorIds,
+              authors: repositoryBehaviorMetadata.authors,
+              changelog: repositoryBehaviorMetadata.changelog,
+              // Keep installed behavior metadata
```

Comment:
I don't understand this. Can you move this comment up at the ternary level and reword it?